Resources/Prototypes/Loadouts/Miscellaneous/glasses.yml

```diff
@@ -0,0 +1,66 @@
+# Timers
+- type: loadoutEffectGroup
+  id: JamjarTimer
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:DepartmentTimeRequirement
+      department: Science
+      time: 36000 #10 hours of being a NERD
```

Comment:
jamjar glasses are a librarian item so they really shouldn't be tied to science

Reply:
> jamjar glasses are a librarian item so they really shouldn't be tied to science

would you prefer for me to remove the requirement? I could make it like, twenty minutes of librarian instead (nobody wants to play librarian)
what about the jensen glasses? are those fine?

Reply:
jensen ones are fine. do w/e you like with the jamjar. maybe just make them a librarian only thing idk.